c7n/resources/cw.py

```diff
@@ -963,11 +965,13 @@ def process(self, resources):
         filter_pattern = self.data.get('filter_pattern', '')
         destination_arn = self.data.get('destination_arn')
         distribution = self.data.get('distribution', 'ByLogStream')
+        role_arn = self.data.get('role_arn')
```

Comment:
we should only pass role_arn if its specified

Reply:
yep .. we caught it this morning ... added the case and test for it